extensions/kafka-connect/src/test/java/com/hazelcast/jet/kafka/connect/KafkaConnectCouchbaseIntegrationTest.java

```diff
@@ -133,7 +132,6 @@ public void testReading() throws Exception {
         config.getJetConfig().setResourceUploadEnabled(true);
         LOGGER.info("Creating a job");
         Job job = createHazelcastInstance(config).getJet().newJob(pipeline, jobConfig);
-        assertJobStatusEventually(job, RUNNING);
```

Comment:
I am not sure how this would happen.

The source is a streaming source so it will run indefinitely until the job fails or we cancel the job.

The assertion waits for 2 * ITEM_COUNT items, we have ITEM_COUNT items with `items-` name, and we insert the second batch of items below this line. So I think the assert here is safe and how the test is constructed - insert some items, start job, wait for it to be running, insert more items is good.